File: modul 5/tugas no 5 praktikum pemrograman 5/prak05.py
def biodata(tahun_lahir, namaku, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahun_lahir
    print(f"Perkenalkan Nama Saya: {namaku}")
    print(f"Umur saya: {umur}")
    print(f"Saya adalah angkatan: {tahun_sekarang}")
    print(f"Asal saya dari: {asal}")
tahun_lahir = int(input())
namaku = input()
asal = input()
biodata(tahun_lahir, namaku, asal)

File: modul 5/tugas no 5 praktikum pemrograman 5/prak05.c
#include <stdio.h>
void Biodata(int tahunLahir, char Namaku[30] ,char Asal[15]){
int tahun_sekarang = 2020;
int umur = tahun_sekarang - tahunLahir; 
printf("Perkenalkan Nama Saya :%s \n",Namaku);
printf("umur saya: %d\n",umur);
printf("saya adalah angkatan: %d\n",tahun_sekarang);
printf("asal saya dari: %s",Asal);
}

int main() {
int tahunLahir;
char A[20], B[15];
scanf(" %d",&tahunLahir);
scanf(" %[^\n]%*c",&A);
scanf(" %[^\n]%*c",&B);
Biodata(tahunLahir, A, B);
return 0;
}
 